astropy/io/votable/table.py

```diff
@@ -31,7 +30,6 @@
 VERIFY_OPTIONS = ["ignore", "warn", "exception"]
```

Comment:
I don't really like that the options list is written out twice - both here and in the configuration item definition. Perhaps this definition should be moved to `astropy/io/votable/__init__.py` so that it could be used for defining the configuration item, and `parse()` could then import the constant for validating the user-provided function argument. An alternative would be to not have this constant at all, but getting the allowed values from the configuration item for an error message seems to be more complicated than moving the constant.

Reply:
Okay, I moved it.